ppl/zqd/apiserver/postgresdb.go

```diff
@@ -0,0 +1,80 @@
+package apiserver
```

Comment:
In a subsequent PR, we could rearrange the manager & filedb code so this postgres implementation could live under `ppl/zqd/postgres`.

Reply:
Makes sense